src/js/module/map/map.js

```diff
@@ -23,33 +31,45 @@ const MapView = View.extend({
         });
         this.panorama = this.map.getStreetView();
 
-        google.maps.event.addListener(this.panorama, 'visible_changed', function() {
-          this.app.trigger(Const.Event.VISIBLE_CHANGED, this.panorama.getVisible());
-        }.bind(this));
-      }.bind(this));
-
+        mapEvent(google.maps.event, this.panorama)
+          .addListener('visible_changed', () => {
```

Comment:
Do I understand correctly, that `visible_changed`, `position_changed` and `pov_changed` are events handled by google maps API?